docs/trulens_eval/tracking/logging/where_to_log/log_in_snowflake.md

```diff
@@ -23,24 +23,23 @@ Snowflake SQLAlchemy can be installed from [PyPI](https://pypi.org/project/snowf
 
 ## Connect TruLens to the Snowflake database
 
-Connecting TruLens to a Snowflake database for logging traces and evaluations only requires passing in a snowflake database URL. Below we've included an example where you can fill in your configuration details to have them formatted into a proper database url and connect to TruLens.
+Connecting TruLens to a Snowflake database for logging traces and evaluations only requires passing in Snowflake [connection parameters](https://docs.snowflake.com/developer-guide/python-connector/python-connector-api#connect).
 
-!!! example "Connect TruLens to the Snowflake database"
+!!! example "Connect TruLens to your Snowflake database"
 
     ```python
     from trulens_eval import Tru
-    tru = Tru(database_url=(
-        'snowflake://{user}:{password}@{account_identifier}/'
-        '{database}/{schema}?warehouse={warehouse}&role={role}'
-    ).format(
-        user='<user>',
-        password='<password>',
-        account_identifier='<account-identifer>',
-        database='<database>',
-        schema='<schema>',
-        warehouse='<warehouse>',
-        role='<role>'
-    ))
+    tru = Tru(
+        name="MyApp",
+        snowflake_connection_parameters={
+            account: "<account>",
+            user: "<user>",
+            password: "<password>",
+            database: "<database>",
+            warehouse: "<warehouse>",
+            role: "<role>",
+        },
```

Comment:
We need to support another workflow to go through snowsql config. Look into https://docs.snowflake.com/en/developer-guide/snowflake-python-api/snowflake-python-connecting-snowflake it supports two things. This would mean, we have to `import snowflake.connector`

Reply:
yeah so I comment on this on the description of the PR. PTAL:
https://github.com/truera/trulens/pull/1314#:~:text=Instead%20of%20giving,it%20could%20change.